### ISSUE TITLE
How to get  or convert the corrent value of datetime？

### ISSUE THREAD

**Issue by Songjiadong** [Azure/hdinsight-phoenix-sharp#19]
I use the phoenix to create a table (eg:user table) including the property using date type,then get this value using PrepareAndExecuteRequestAsync method.But I find the value's Rep is integer type not JavaSqlDate or JavaSqlTime or JavaUtilDate and its columnclassname is "java.sql.Date",so How to get  or convert the corrent value of datetime?  Could you give me an an example? Thanks.

**Comment by egeaydin** [comment]
Hey mate,

I know it's been a little while you posted this but in case anybody stumbles on this, the solution is this:
```c#
 public DateTime ConvertToCSharpType(TypedValue value)
        {
            var startDate = new DateTime(1970, 1, 1, 0, 0, 0, DateTimeKind.Utc);

            if (value.Type == Rep.Integer)
                return startDate.AddDays(value.NumberValue);
            else if (value.Type == Rep.Long)
                return startDate.AddMilliseconds(value.NumberValue);
            else
                throw new NotImplementedException($"Cannot convert value type {value.Type} to DateTime because it is not implemented");
        }`
                
Basically, I discovered that if the data type from DB is `Date`, then the value returned would be `Rep.Integer` which would indicate a number of days after 1/1/1970. If the date type from DB is `DateTime`, then the value returned is `Rep.Long` which indicates the number of milliseconds after 1/1/1970.
